VERSIONING.md

```
@@ -0,0 +1,31 @@
+
+> Given a version number MAJOR.MINOR.PATCH, increment the:
+>
+> 1. MAJOR version when you make incompatible API changes
+> 2. MINOR version when you add functionality in a backward compatible manner
+> 3. PATCH version when you make backward compatible bug fixes
+
+However, in some areas the line is blurry and controversial, and very occasionally deemed to be impractical in the short-term.  The rest of this document outlines these special areas and our versioning policy on them.
+
+## Errors
+
+At the moment, our errors are largely string based - they are driven by a single concrete implementation of the standard lib `errors` interface [here](./errors/defraError.go#L60).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)